src/handlers.js

```diff
@@ -12,17 +12,19 @@ import TaskbarIcon from './desktop/taskbaricon/taskbaricon';
 import ToDoApp from "./defaultapps/TodoApp/TodoApp";
 import Calculator from "./defaultapps/Calculator/calculator";
 
-
 class runningTasks
 {
-    constructor(runningId,appData)
+    constructor(runningId,appData,code)
```

Comment:
I think this place doesnt require any changes, as this is meant to be a metadata for running applications.

Reply:
Okay, I'll get rid of this.